docs/src/main/sphinx/object-storage/file-system-cache.md

```diff
@@ -134,6 +134,17 @@ enable and configure caching for the specific catalogs.
     smaller values are less efficient since they result in more individual downloads.
 :::
 
+## Monitoring
+
+The cache exposes the
+[Alluxio JMX client metrics](https://docs.alluxio.io/ee-da/user/stable/en/reference/Metrics-List.html#client-metrics)
+under the `org.alluxio` package. Not all the JMX metrics on the workers are
+propagated to the coordinator, and are not available in the JMX connector.
+They must be collected on the workers directly.
+
+The cache code uses OpenTelemetry tracing, and the spans can be exported with
```

Comment:
```suggestion
The cache code uses [OpenTelemetry tracing](/admin/opentelemetry).
```

I dont think we need the rest of the sentence .. since there is nothing to do in terms of "exporting the spans" ..